File: mario.c
#include <cs50.h>
#include <stdio.h>

void make_row(int brick);

int main(void)
{
    int h;
    do
    {
        h = get_int("What's the hight you want? ");
    }
    while (h < 1 || h > 8);

    make_row(h);
}
void make_row(int height)
{
    int p = height;
    for (int i = 0; i < height; i++)
    {
        for (int j = 0; j < p - 1; j++)
        {
            printf(" ");
        }
        for (int k = 0; k < i + 1; k++)
        {
            printf("#");
        }
        printf("\n");
        p--;
    }
}

File: Week 6 - Python/mario.py
from cs50 import get_int


def makePyramid(height):
    for i in range(height):
        for j in range(height-i-1):
            print(" ", end="")
        for k in range(i+1):
            print("#", end="")
        print()


stop = True
while (stop):
    height = get_int("height? ")
    if (height > 0 and height < 9):
        stop = False

makePyramid(height)
